crates/neon/src/types_impl/buffer/types.rs

```diff
@@ -475,7 +496,11 @@ impl<T: Binary> Managed for JsTypedArray<T> {
     }
 }
 
-impl<T: Binary> TypedArray for JsTypedArray<T> {
+impl<T> TypedArray for JsTypedArray<T>
+where
+    T: Binary + Copy,
```

Comment:
Do you think it makes sense to make `Copy` a supertype of `Binary` instead of adding the bound here?

Reply:
I originally did… you asked me to remove it 😝

Reply:
😆 It made sense in the last PR, but then once we introduced a `Value` constraint, the separation was less valuable.

Reply:
I dig it 👍